### PR TITLE
document more known issues

### LINKED ISSUE #1194: Cannot create cluster in on ubuntu on IBM PPC64LE machine
**What happened**:
Tried to create a cluster with kind create cluster and get:

Creating cluster "kind" ...
 ✓ Ensuring node image (kindest/node:v1.16.3) 🖼
 ✓ Preparing nodes 📦
 ✗ Writing configuration 📜
ERROR: failed to create cluster: failed to generate kubeadm config content: failed to get kubernetes version from node: failed to get file: command "docker exec --privileged kind-control-plane cat /kind/version" failed with error: exit status 1
**What you expected to happen**:
creating cluster
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment:**

- kind version: (use `kind version`):kind v0.6.1 go1.13.4 linux/ppc64le
- Kubernetes version: (use `kubectl version`):
no
- Docker version: (use `docker info`):
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 3
Server Version: 18.06.1-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: nvidia runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 468a545b9edcd5932818eb9de8e72413e616e86e
runc version: 69663f0bd4b60df09991c08812a60108003fa340
init version: fec3683
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.15.0-60-generic
Operating System: Ubuntu 18.04.3 LTS
OSType: linux
Architecture: ppc64le
CPUs: 160
Total Memory: 255.3GiB
Name: PWRAI-2
ID: QZYB:Z2XW:ZBXP:AWJS:HE3S:MIM6:HVFO:UYO3:54OA:3OE4:JFO7:AHTW
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

- OS (e.g. from `/etc/os-release`):
Linux 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:54:48 UTC 2019 ppc64le ppc64le ppc64le GNU/Linux

### PR DESCRIPTION
**Pull request by BenTheElder**
fixes #1194 (document known issue / how to handle this) and does some minor cleanup

### CODE REVIEW

**Comment by aojea** [comment]
_how do you see the netlify previews_ ???

**Comment by BenTheElder** [comment]
> how do you see the netlify previews ???

the url is 100% predictable, just has the PR # in it.

PR 1197 -> https://deploy-preview-1197--k8s-kind.netlify.com/

`k8s-kind` is the netlify instance. `deploy-preview` explains itself.

there _are_ other previews, but that's really the only one you need.
:man_shrugging: 

in this case, just the raw markdown should be pretty self explanatory

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -17,20 +17,21 @@ It may additionally be helpful to:
 - reach out and ask for help in [#kind] on the [kubernetes slack]
 
 ## Contents
-* [Failures involving mismatched kubectl versions](#failures-involving-mismatched-kubectl-version)
+* [kubectl version skew](#kubectl-version-skew)
```
Comment (aojea) [inline site/content/docs/user/known-issues.md]: should start with caps for uniformity `Kubect Version Skew` :/

Comment (BenTheElder) [reply site/content/docs/user/known-issues.md]: I've never seen kubectl capitalized, but your version is missing the `l` :stuck_out_tongue: 

`kubectl` is the correct name imo, it's a binary. similar to `systemd`

https://kubernetes.io/docs/reference/kubectl/overview/ has a singla capital kubectl for starting the sentence. when a name in a heading it is lower case unlike eg `Kubernetes`.

Comment (BenTheElder) [reply site/content/docs/user/known-issues.md]: with `kind` I've given up and generally use same case, with `KIND` or `KinD` used to help disambiguate (or `sigs.k8s.io/kind`) but for tools with better recognition I prefer to follow upstream..

Comment (aojea) [reply site/content/docs/user/known-issues.md]: I just was doing a diff against the header it links and the others links underneath it ... just bikeshedding a bit before taking some 💤 
👋

**Pull request merged**